uefi-sct/SctPkg/Library/SctLib/DevicePath.c

```diff
@@ -16,7 +16,8 @@
 
 #include "SctLibInternal.h"
 #include "SctLib.h"
-
+#include "Protocol/StandardTestLibrary.h"
+#include "Library/EfiTestLib.h"

```

Comment:
Just curious why we need to include EfiTestLib.h?  
If we need to include EfiTestLib, please add it to SctLib.inf file instead.

Reply:
EfiTestLib.h is needed for gtbs to use at `gtBS->FreePool (DevicePathStr)`
Similarly including it directly with sctlib is causing redeclaration issues, so I limited the scope of the inclusion by just including it in devicepath.c

Reply:
In this case, you should add UefiBootServicesTableLib to SctLib.inf instead.